LibreNMS/Validations/Scheduler.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Scheduler.php
+ *
+ * -Description-
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * @link       https://www.librenms.org
+ */
+
+namespace LibreNMS\Validations;
+
+use Illuminate\Support\Facades\Cache;
+use LibreNMS\Validator;
+
+class Scheduler extends BaseValidation
+{
+    /**
+     * Validate this module.
+     * To return ValidationResults, call ok, warn, fail, or result methods on the $validator
+     *
+     * @param  Validator  $validator
+     */
+    public function validate(Validator $validator): void
+    {
+        if (! Cache::has('scheduler_working')) {
+            $validator->fail('Scheduler is not running!',
+                'cp /opt/librenms/librenms.nonroot.cron /etc/cron.d/librenms');
```

Comment:
This fix won't be correct if they have disabled some of the cron for the dispatcher.

Maybe sed?

Reply:
or a separate cron file.

Reply:
> This fix won't be correct if they have disabled some of the cron for the dispatcher.
> 
> Maybe sed?

TLDR; no hacks with sed plz. The source file contains the row for this scheduler, we can't really do much about users editing files, there is enough of warnings in place for that already?

Since this could/should replace all/most of the cron file in the future, maybe we can do separate ones.
I'll fix.